docs/how_to_publish_to_cmr_with_earthdata_varinfo.ipynb

```diff
@@ -0,0 +1,413 @@
+    "* [M2I1NXASM](https://cmr.uat.earthdata.nasa.gov/search/collections.umm_json?concept-id=C1256535511-EEDTEST)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Authorization:\n",
+    "\n",
+    "* Launchpad or Bearer tokens must be used in order query and publish to CMR.\n",
+    "* Bearer tokens are tokens that contain the header `Bearer` before the token\n",
+    "    * For example: `Bearer tHiS iS mY tOKen`\n",
+    "* Launchpad tokens are tokens that do **NOT** contain any headers\n",
+    "    * For example: `tHiS iS mY tOKen`\n",
+    "To request a Launchpad Token visit:\n",
+    "* [Launchpad Authentication User's Guide](https://wiki.earthdata.nasa.gov/display/CMR/Launchpad+Authentication+User%27s+Guide)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Publish UMM-Var records for **GLDAS_NOAH10_3H** with `generate_collections_umm_var`\n",
+    "\n",
+    "`generate_collections_umm_var` is a wrapper function that combines the functionalities in `varinfo.cmr_search`, the `VarInfoFromNetCDF4` class and `varinfo.umm_var` to create and publish UMM-Var entries to CMR."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Update `auth_header` to include your Bearer (e.g. `'Bearer tHiS iS mY tOKen'`) or Launchpad token (e.g. `tHiS iS mY tOKen`)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth_header = 'Header + Token or Only token'"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Update the `collection_concept_id` to the **GLDAS_NOAH10_3H** concept-id for the EEDTEST provider.\n",
+    "* This can be updated to any concept-id for any provider"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "collection_concept_id_gldas = 'C1256543837-EEDTEST'"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Import `generate_collection_umm_var` from `varinfo.generate_umm_var`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from varinfo.generate_umm_var import generate_collection_umm_var"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`generate_collection_umm_var` will:\n",
+    "\n",
+    "* Download the most recent granule for **GLDAS_NOAH10_3H**\n",
+    "* Generate the UMM-Var records for this granule\n",
+    "* Publish these records to CMR if `publish=True` and return a list of published variable concept-ids\n",
+    "(e.g. `['V1259971755-EEDTEST', 'V1259971757-EEDTEST' ...]`)"
```

Comment:
What about if there is an error?

Seems like a user would want to know what that looks like, and how we'd recommend fixing any issue. (Not sure if this is the best approach, but potentially using the nitty-gritty function targeted to deal with the single variable, allowing local resolution of the UMM-Var record, maybe?)

Reply:
* [ ] Add example including errors